### ISSUE TITLE
Windows builds are missing the .exe extension

### ISSUE THREAD

**Issue by thorgeirk11** [webbertakken/unity-actions#35]
The output of a StandaloneWindows64 build is missing the .exe extension of the file. 

My setup:
```
- name: Make Unity Client
  uses: webbertakken/unity-builder@v0.7
  env:
      UNITY_EMAIL:    ${{ secrets.UNITY_EMAIL }}
      UNITY_PASSWORD: ${{ secrets.UNITY_PASSWORD }}
      UNITY_SERIAL:   ${{ secrets.UNITY_SERIAL }}
  with:
      unityVersion: 2018.4.10f1
      targetPlatform: StandaloneWindows64
      projectPath: Client/
```

![image](https://user-images.githubusercontent.com/2693474/72733618-190f8b80-3b90-11ea-8f18-3bc672bf2365.png)

The good thing is if I just manually add the `.exe` extension to the `StandaloneWindows64` file everything works fine.

**Comment by GabLeRoux** [comment]
I've had this problem as well in https://gitlab.com/gableroux/unity3d-gitlab-ci-example/

Here's how I solved this:  
https://gitlab.com/gableroux/unity3d-gitlab-ci-example/blob/c4f9d42d/Assets/Scripts/Editor/BuildCommand.cs#L82-90

**Comment by webbertakken** [comment]
This is fixed and released in [builder v0.8](https://github.com/webbertakken/unity-builder/releases/tag/v0.8).
